src/snovault/resource_views.py

```diff
@@ -277,3 +277,11 @@ def item_view_edit(context, request):
         )
 
     return properties
+
+
+@view_config(context=Item, permission='edit', request_method='GET',
+             name='form', decorator=etag_tid)
+def item_view_form(context, request):
+    properties = item_view_edit(context, request)
+    properties['@type'] = context.jsonld_type()
```

Comment:
I'm not aware of any, but I wasn't sure what code out there might be relying on the edit frame to only return properties that can be PUT back. I guess it would break submitting the edit-json form.